src/components/footer/footer.scss

```diff
@@ -0,0 +1,7 @@
+@import '../variables';
```

Comment:
why you need variables here? What variable do you use from there?

Reply:
I agree, it is not necessary here and will be delete